### ISSUE TITLE
SC2016, expansion in single quotes shouldn't trigger for test -v

### ISSUE THREAD

**Issue by koalaman** [koalaman/shellcheck#873]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2016
- My shellcheck version (`shellcheck --version` or "online"): 
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
[ -v 'foo[$var]' ]
```

#### Here's what shellcheck currently says:

```
[ -v 'foo[$var]' ]
     ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.
 ```
#### Here's what I wanted or expected to see:

Nothing.

**Comment by koalaman** [comment]
Fixed in 1fd9b47
